pkg/transformer/kubernetes/k8sutils_test.go

```diff
@@ -67,12 +67,60 @@ func TestCreateService(t *testing.T) {
 
 	// Test the creation of the service
 	svc := k.CreateService("foo", service, objects)
+
 	if svc.Spec.Ports[0].Port != 123 {
 		t.Errorf("Expected port 123 upon conversion, actual %d", svc.Spec.Ports[0].Port)
 	}
 }
 
 /*
+	Test the creation of a service with a memory limit
+*/
+func TestCreateServiceWithMemLimit(t *testing.T) {
+
+	// An example service
+	service := kobject.ServiceConfig{
+		ContainerName: "name",
+		Image:         "image",
+		Environment:   []kobject.EnvVar{kobject.EnvVar{Name: "env", Value: "value"}},
+		Port:          []kobject.Ports{kobject.Ports{HostPort: 123, ContainerPort: 456, Protocol: api.ProtocolTCP}},
+		Command:       []string{"cmd"},
+		WorkingDir:    "dir",
+		Args:          []string{"arg1", "arg2"},
+		Volumes:       []string{"/tmp/volume"},
+		Network:       []string{"network1", "network2"}, // not supported
+		Labels:        nil,
+		Annotations:   map[string]string{"abc": "def"},
+		CPUSet:        "cpu_set",            // not supported
```

Comment:
Ahh, OK than.
I just saw that we are loading into kobject, but didn't realise that we do nothing with it afterwords. That is confusing :-)